pkgs/stdenv/linux/make-bootstrap-tools.nix

```diff
@@ -106,7 +106,7 @@ in with pkgs; rec {
         cp -d ${patch}/bin/* $out/bin
         cp ${patchelf}/bin/* $out/bin
 
-        cp -d ${gnugrep.pcre.out}/lib/libpcre*.so* $out/lib # needed by grep
+        cp -d ${gnugrep.pcre.out}/lib/libpcre*.so* $out/lib || true # needed by grep
```

Comment:
Why are the changes in this file needed?

Reply:
Interesting that it is only showing up on one architecture. What's the error?

I see that buildroot doesn't apply any patches to PCRE but they do apply some cross-compilation patches to all libtool`s ltmain.sh files. Maybe those could contain a clue.

Reply:
https://gist.github.com/volth/4cfca989bec2bac5c816b50b47599bcf#file-mips64el-pcre-build-log-L131-L132

Reply:
Ah, there's an even earlier error about not being able to build shared libraries:
````
checking whether the mips64el-unknown-linux-gnu-gcc linker (mips64el-unknown-linux-gnu-ld -m elf) supports shared libraries... /nix/store/s0qmjv3v28c0nni261fq08rpjiriv9ra-mips64el-unknown-linux-gnu-binutils-2.30/bin/mips64el-un
known-linux-gnu-ld: unrecognised emulation mode: elf
Supported emulations: aix5ppc aix5rs6 aixppc aixrs6 alpha alphavms arcv2elf arcv2elfx arcelf arcelf_prof arclinux arclinux_nps arclinux_prof arm_epoc_pe arm_wince_pe armaoutb armaoutl armcoff armelf armelf_fbsd armelf_fuchsia a
rmelf_linux armelf_linux_eabi armelf_nacl armelf_nbsd armelf_phoenix armelf_vxworks armelfb armelfb_fbsd armelfb_fuchsia armelfb_linux armelfb_linux_eabi armelfb_nacl armelfb_nbsd armnbsd armnto armpe armsymbian avr1 avr2 avr25
 avr3 avr31 avr35 avr4 avr5 avr51 avr6 avrxmega1 avrxmega2 avrxmega3 avrxmega4 avrxmega5 avrxmega6 avrxmega7 avrtiny coff_i860 coff_sparc crisaout criself crislinux d10velf d30v_e d30v_o d30velf delta68 elf32_dlx elf32_i860 elf
32_i960 elf32_sparc elf32_sparc_sol2 elf32_sparc_vxworks elf32_spu elf32_tic6x_be elf32_tic6x_le elf32_tic6x_linux_be elf32_tic6x_linux_le elf32_tic6x_elf_be elf32_tic6x_elf_le elf32am33lin elf32bfin elf32bfinfd elf32cr16 elf32
cr16c elf32crx elf32epiphany elf32epiphany_4x4 elf32fr30 elf32frv elf32frvfd elf32ft32 elf32i370 elf32ip2k elf32iq10 elf32iq2000 elf32lm32 elf32lm32fd elf32lppc elf32lppclinux elf32lppcnto elf32lppcsim elf32m32c elf32mb_linux e
lf32mbel_linux elf32mcore elf32mep elf32metag elf32microblazeel elf32microblaze elf32moxie moxiebox elf32mt elf32or1k elf32or1k_linux elf32ppc elf32ppc_fbsd elf32ppclinux elf32ppcnto elf32ppcsim elf32ppcvxworks elf32ppcwindiss
elf32lriscv elf32rl78 elf32rx elf32tilegx elf32tilegx_be elf32tilepro elf32vax elf32visium elf32xc16x elf32xc16xl elf32xc16xs elf32xstormy16 elf32xtensa elf_i386 elf_i386_be elf_i386_chaos elf_i386_fbsd elf_i386_ldso elf_i386_n
acl elf_i386_sol2 elf_i386_vxworks elf_iamcu elf_s390 960 960coff h8300 h8300elf h8300elf_linux h8300h h8300helf h8300helf_linux h8300hn h8300hnelf h8300s h8300self h8300self_linux h8300sn h8300snelf h8300sx h8300sxelf h8300sxe
lf_linux h8300sxn h8300sxnelf h8500 h8500b h8500c h8500m h8500s hp300bsd hp3hpux hppaelf hppalinux hppanbsd hppaobsd i386aout i386beos i386bsd i386coff i386go32 i386linux i386lynx i386mach i386moss i386msdos i386nbsd i386nto i3
86nw i386pe i386pe_posix lnk960 m32relf m32relf_linux m32rlelf m32rlelf_linux m68hc11elf m68hc11elfb m68hc12elf m68hc12elfb m68k4knbsd m68kaout m68kaux m68kcoff m68kelf m68kelfnbsd m68klinux m68knbsd m68kpsos m88kbcs mcorepe mn
10200 mn10300 msp430elf msp430X nds32elf nds32elf16m nds32elf_linux nds32belf nds32belf16m nds32belf_linux news ns32knbsd nios2elf nios2linux pc532macha pdp11 pjelf pjlelf ppclynx ppcmacos ppcnw ppcpe pruelf riscix score3_elf s
core7_elf sh shelf shelf32 shelf32_linux shelf32_nbsd shelf_fd shelf_linux shelf_nbsd shelf_nto shelf_uclinux shelf_vxworks shl shlelf shlelf32 shlelf32_linux shlelf32_nbsd shlelf_fd shlelf_linux shlelf_nbsd shlelf_nto shlelf_v
xworks shlsymbian shpe sparcaout sparclinux sparcnbsd st2000 sun3 sun4 tic30aout tic30coff tic3xcoff tic3xcoff_onchip tic4xcoff tic54xcoff tic80coff v850 v850_rh850 vanilla vax vaxnbsd vsta w65 xgateelf z80 z8001 z8002 aarch64e
lf aarch64elf32 aarch64elfb aarch64elf32b aarch64cloudabi aarch64cloudabib aarch64fbsd aarch64fbsdb aarch64linux aarch64linuxb aarch64linux32 aarch64linux32b elf32_x86_64 elf32_x86_64_nacl elf32b4300 elf32bmip elf32bmipn32 elf3
2bsmip elf32btsmip elf32btsmip_fbsd elf32btsmipn32 elf32btsmipn32_fbsd elf32ebmip elf32ebmipvxworks elf32elmip elf32elmipvxworks elf32l4300 elf32lmip elf32lr5900 elf32lr5900n32 elf32lsmip elf32ltsmip elf32ltsmip_fbsd elf32ltsmi
pn32 elf32ltsmipn32_fbsd elf32mipswindiss elf64_aix elf64_ia64 elf64_ia64_fbsd elf64_ia64_vms elf64_s390 elf64_sparc elf64_sparc_fbsd elf64_sparc_sol2 elf64alpha elf64alpha_fbsd elf64alpha_nbsd elf64bmip elf64btsmip elf64btsmip
_fbsd elf64hppa elf64lppc elf64lriscv elf64ltsmip elf64ltsmip_fbsd elf64mmix elf64ppc elf64ppc_fbsd elf64rdos elf64tilegx elf64tilegx_be elf_l1om elf_l1om_fbsd elf_k1om elf_k1om_fbsd elf_x86_64 elf_x86_64_cloudabi elf_x86_64_fb
sd elf_x86_64_nacl elf_x86_64_sol2 hppa64linux i386pep mmo shelf64 shelf64_nbsd shlelf64 shlelf64_nbsd
no
````

Reply:
Buildroot says:
````
checking whether the /home/tmtynkky/cross/buildroot/output/host/bin/mips64-buildroot-linux-uclibc-gcc linker (/home/tmtynkky/cross/buildroot/output/host/bin/mips64-buildroot-linux-uclibc-ld -m elf64btsmip) supports shared libraries... yes
````